### ISSUE TITLE
Add explanatory text to 'Shipping Labels' page

### ISSUE THREAD

**Issue by v18** [Automattic/woocommerce-services#1116]
On this page:

<img width="966" alt="screen shot 2017-08-07 at 1 55 42 pm" src="https://user-images.githubusercontent.com/11487924/29039150-2052b960-7b78-11e7-8a68-1bcaedfbd2ac.png">

We should have text explaining what the credit card will be used for:

"Enter a credit card so that you can buy postage from USPS"

**Comment by v18** [comment]
So, looks like that text already exists but users are not reading it :). Currently it says:

> Use your credit card on file to pay for the labels you print or add a new one.

<img width="966" alt="29039150-2052b960-7b78-11e7-8a68-1bcaedfbd2ac" src="https://user-images.githubusercontent.com/11487924/29819386-3a03e11a-8cc9-11e7-94aa-353482bfef99.png">

@olaolusoga do you think we should close this issue until it comes up again, or come up with a better location for that text, or some other solution for this problem?

**Comment by olaolusoga** [comment]
> do you think we should close this issue until it comes up again, or come up with a better location for that text, or some other solution for this problem?

I say we close since it really low hanging fruit, and the text is already there. We should keep a mental note of it though, since we'll need to touch it down the road per convo here #1115

**Comment by dechov** [comment]
We could maybe reword it to front-load the emphasis: "To purchase labels, use your credit card on file or add a new one."

**Comment by v18** [comment]
@dechov -  I like it. Would you be able to open a PR with that change?
